modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonAbstractConnexionServerCodegen.java

```diff
@@ -201,6 +201,7 @@ public void processOpts() {
         }
         supportingFiles.add(new SupportingFile("__main__.mustache", packagePath(), "__main__.py"));
         supportingFiles.add(new SupportingFile("util.mustache", packagePath(), "util.py"));
+        supportingFiles.add(new SupportingFile("typing_patch.mustache", packagePath(), "typing_patch.py"));
```

Comment:
Suggestion: what about naming it as `typing_utils.py` instead?

Patch sounds like the code is more like a workaround and not production ready

Reply:
Yes, it's is better with `typing_utils.py`